packages/gatsby/src/redux/actions/public.js

```diff
@@ -364,17 +364,17 @@ ${reservedFields.map(f => `  * "${f}"`).join(`\n`)}
 
   if (invalidPathSegments.length > 0) {
     const truncatedPath = truncatePath(page.path)
-    report.panicOnBuild({
-      id: `11331`,
-      context: {
-        path: page.path,
-        invalidPathSegments,
-
-        // we will only show truncatedPath in non-production scenario
-        isProduction: process.env.NODE_ENV === `production`,
-        truncatedPath,
-      },
-    })
+    report.warn(
```

Comment:
why warn at all? The user wouldn't need to do anything to fix this & (probably) wouldn't ever care that we'd truncated the path. Maybe just remove the error altogether?

Reply:
@KyleAMathews Because this way the user atleast gets told _what_ the truncated url is.